clients/da-vinci-client/src/main/java/com/linkedin/davinci/client/AvroGenericDaVinciClient.java

```diff
@@ -114,7 +114,7 @@ protected boolean removeEldestEntry(Map.Entry<Schema, GenericRecord> eldest) {
   private RecordDeserializer<K> keyDeserializer;
   private AvroGenericReadComputeStoreClient<K, V> veniceClient;
   private StoreBackend storeBackend;
-  private static ReferenceCounted<DaVinciBackend> daVinciBackend;
+  private static volatile ReferenceCounted<DaVinciBackend> daVinciBackend;
```

Comment:
Can you please explain why this needs to be `volatile`? From what I can see, it's used only inside synchronized section, which should be sufficient to make any change globally visible.

Reply:
Will remove, leftover code from experimentation while debugging.